samples/gen-js/test/test.js

```diff
@@ -0,0 +1,35 @@
+const assert = require("assert");
```

Comment:
These tests look real nice - can we make them run as part of `make test` in the wag repo?